app/models/opensearch.rb

```diff
@@ -66,21 +67,14 @@ def multisearch
 
   def matches
     m = []
-    if @params[:q].present?
-      m << {
-      multi_match: {
-        query: @params[:q].downcase
-        }
-      }
-    end
```

Comment:
I think right now `q` is only matching in titles and contributors from the `multimatch` block. This entire `if` block, which confusingly is a `multi_match` but done in the `match` block (I promise it makes sense!) needs to be added back in.

You can see this by connecting this branch to the dev1 open search and entering this GraphQL Query:
```
{
  search(searchterm: "Standards") {
    hits
    records {
      title
      subjects {
        kind
        value
      }
    }
  }
}
```

Note only one hit. Add the lines back in, rerun the search, and you'll see 6 hits and that there were matches this time in additional fields that should be searched when doing keyword searches. The method naming is definitely not helpful here and we can consider how to better address that as we work more with this together.

Reply:
@JPrevost Ha, I knew that was there for a reason. I added it back in and confirmed the correct behavior by successfully searching metadata in non-title/contributors fields with `q`.

I didn't make a regression test for this because it seemed like it would require generating a ton of cassettes to cover every case, but I could just add a test that confirms a successful search for, e.g., an ISBN in `q`.